### ISSUE TITLE
Deprecated syntax

### ISSUE THREAD

**Issue by pgrayove-mcpl** [backdrop-contrib/smart_ip#7]
This error message displayed on my development site:
Deprecated: Using ${var} in strings is deprecated, use {$var} instead in /app/modules/contrib/smart_ip/smart_ip.module on line 402 Deprecated: Using ${var} in strings is deprecated, use {$var} instead in /app/modules/contrib/smart_ip/smart_ip.module on line 418

I changed the two lines mentioned from:
`$output .= "${degrees}&deg; ${minutes}' ${seconds}\" ";`
to:
`$output .= "{$degrees}&deg; {$minutes}' {$seconds}\" ";`

Seems to have fixed the error.

**Comment by swampopus** [comment]
Yeah, I'm not sure why they used the curly braces at all, honestly.  I just changed it to this:

`$output .= "$degrees&deg; $minutes' $seconds\" ";`

since you can have normal variables in a double-quoted string.

Anyway, fixed in the current commit.
